final_task/rss_reader/string_operations.py

```diff
@@ -0,0 +1,63 @@
+from html.parser import HTMLParser
+
+
+def cut_string_to_length_with_space(basic_str, length):
+    """function that cut given string to the list of strings with a given max length"""
+    strings_list = []
+    string = ''
+    for i, word in enumerate(basic_str.split()):
+        string += word + ' '
+        if i + 1 < len(basic_str.split()):
+            next_word = basic_str.split()[i+1]
+        else:
+            next_word = ''
+            strings_list.append(string)
+        if len(string + next_word) >= length:
+            strings_list.append(string)
+            string = ''
+    return strings_list
+
+
+def make_string_readable(basic_str):
+    h = HTMLParser()
```

Comment:
Still one-letter variable here. It is better to avoid them at any given time, even in such simple and small functions

Reply:
Fixed